payments-core/src/main/java/com/stripe/android/networking/StripeApiRepository.kt

```diff
@@ -157,6 +161,12 @@ class StripeApiRepository @JvmOverloads internal constructor(
 
     init {
         fireFraudDetectionDataRequest()
+
+        CoroutineScope(Dispatchers.IO).launch {
```

Comment:
is this change related to the PR?

Reply:
Yes, we need to create a file on disk to store responses into the cache

Reply:
Should we use the injected `workContext` for this coroutine?

Reply:
Should we use the injected `workContext` for this coroutine?

Reply:
I think it makes sense to keep it as IO, since it is technically an IO scope. Does that make sense or does `workContext` make more sense here

Reply:
I’d like to see it injected instead of set internally. Therefore, using `workContext` (which is an IO context), makes more sense to me.